docs/building-a-form/available-components.md

```diff
@@ -348,18 +414,25 @@ When you build a form with more than one chapter (shown by the segments in a pro
 
 #### Usage guidelines
 
-Break into separate topic?
+In general, the review page renders the form data in review mode automatically. There are, however, some specific options you can pass to the form config to customize certain review functionality.
+
+This property is nested directly under `uiSchema`:
+- `'ui:reviewWidget'`: takes a widget component to render on the review page for that field. Default review widgets are automatically rendered, so only use this if you need to customize the review widget that is used.
+
+These properties are nested under `uiSchema: { `ui:options`: {} }`:
+- `hideOnReview`: Hides the specified field on the review page; takes a `boolean`
+- `hideOnReviewIfFalse`: Hides the specified field on the review page when the field value is `false`; takes a `boolean`
+- `keepInPageOnReview`: Used when you have an array field that should not be pulled out of the page its in and shown separately on the review page; takes a `boolean`
```

Comment:
I'm still not clear on the meaning of this sentence.

Reply:
I'm working on this description in https://github.com/usds/us-forms-system/pull/175/files#diff-7f83b150433f519ec46f0f3f789b3f50R233, should have something 🔜. I can update this file with the results later on.

Reply:
Opened an issue to take care of this: https://github.com/usds/us-forms-system/issues/180.